lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
+  cmd.desc 'Docker tag for application container'
+  cmd.arg_name 'TAG'
+  cmd.flag [:tag]
+end
+
+def add_target_flag(cmd)
+  cmd.desc 'Deployment target to use, e.g. production_web'
+  cmd.arg_name 'TARGET'
+  cmd.flag [:t, :target], type: Symbol, required: true
+end
+
+def add_instance_flag(cmd)
```

Comment:
if `add_instance_flag` ends up adding both the instance and target flags, it's not really intuitive to a reader